src/lesson03/Task06.java

```diff
@@ -0,0 +1,16 @@
+package lesson03;
+
+import java.util.Scanner;
+
+public class Task06 {
+    public static void main(String[] args) {
+        Scanner scan = new Scanner(System.in);
+        System.out.println("Input:");
+        Double n = scan.nextDouble();
+        double fact = 1;
```

Comment:
тут не обовязково використовувати клас обгортку
```suggestion
        double n = scan.nextDouble();
        double fact = 1;
```